newsfragments/README.md

```diff
@@ -0,0 +1,24 @@
+This directory collects "newsfragments": short files that each contain
+a snippet of ReST-formatted text that will be added to the next
+release notes. This should be a description of aspects of the change
+(if any) that are relevant to users. (This contrasts with the
+commit message and PR description, which are a description of the change as
+relevant to people working on the code itself.)
+
+ Each file should be named like `<ISSUE>.<TYPE>.rst`, where
+`<ISSUE>` is an issue number, and `<TYPE>` is one of:
+
+* `feature`
+* `bugfix`
+* `doc`
+* `misc`
+
+So for example: `123.feature.rst`, `456.bugfix.rst`
+
+If the PR fixes an issue, use that number here. If there is no issue,
```

Comment:
As in, use it in the name, or mention it in the file?

Reply:
Also, what if a PR fixes several issues?

Reply:
> As in, use it in the name, or mention it in the file?

In the filename.

> Also, what if a PR fixes several issues?

Make several news fragments (use your judgement).

Reply:
Ah, I just realized that issues and PRs are the same for github, and share identifiers. I guess it doesn't really matter then.